hints/hints.go

```diff
@@ -0,0 +1,76 @@
+package hints
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/manifoldco/torus-cli/ui"
+)
+
+var commandHints map[string][]string
+
+// Init generates the commandHints map
+func Init() {
+	commandHints = map[string][]string{
+		"allow": {
+			"Grant additional access to a team or role using `torus allow`",
+		},
+		"context": {
+			"Your linked organization and project are found in .torus.json after running `torus link`",
+		},
+		"deny": {
+			"Restrict access for a team or role using `torus deny`",
```

Comment:
This is only true, if they've already been given access to something -- so we need to clarify that point, otherwise, this sounds like users have access to everything by default.

Which calls into question the whole paradigm regarding deny.